app/index.js

```diff
@@ -2,14 +2,16 @@
 require('dotenv').config();
 
 import config from './config';
-import Server from './services/server';
+import initServer from './services/server';
 
 var options = {
   connection: config.connection
 };
 
 // Start API server
-var server = new Server(options);
-server.start(() => {
-  // Started.
+initServer(options, (err, server) => {
```

Comment:
As I explained in the commit message, plugin registration is asynchronous. We have to wait till auth to be registered, then register routes with auth, then start the server.